requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+pylint==2.6.0
+pytest==6.1.0
+# anki==2.1.34 doesn't have RemoteServer in sync.py
```

Comment:
I'm aware there are "breaking changes" in recent Anki versions. I'll probably work on them when those versions reach Arch Linux repos (that's how I've installed Anki, currently).

Reply:
Also using Arch linux, we have a lot in common.